### ISSUE TITLE
pact-jvm-provider-maven: Stray trailing " in POST body of request generated in verify step. 

### ISSUE THREAD

**Issue by SushmaShyam** [DiUS/pact-jvm#199]
I am trying to use Pact provider maven plugin for verifying my provider. Below is a json snippet of my pact file.

Below is a json snippet from my pact file

```
      "request": {
        "method": "post",
        "path": "/query",
        "headers": {
          "Content-Type": "application/x-www-form-urlencoded"
        },
        "body": "to=1288781250000"
      }
```

When I run the verification step by running mvn au.com.dius:pact-jvm-provider-maven_2.11:verify, my provider service gets a trailing quote in its request body. 

The request body is ``` to=1288781250000" ``` instead of just  ``` to=1288781250000 ``` .

**Comment by SushmaShyam** [comment]
Is there any other information that I can provide with regard to this bug? Will be happy to do so

**Comment by uglyog** [comment]
Hopefully it should be easy to replicate.

**Comment by uglyog** [comment]
I am unable to replicate this. I'm using your snippet, and I have dumped the request data I'm receiving.

    [main] INFO ratpack.server.RatpackServer - Starting server...
    [main] INFO ratpack.server.RatpackServer - Building registry...
    [main] INFO ratpack.server.RatpackServer - Ratpack started (development) for http://localhost:5050
    path=query
    contentType=application/x-www-form-urlencoded
    headers=[Accept-Encoding:gzip,deflate, Content-Type:application/x-www-form-urlencoded, Host:localhost:5050, Connection:Keep-Alive, Content-Length:16, User-Agent:Apache-HttpClient/4.5.1 (Java/1.8.0_66)]
    body=[to=1288781250000]

You can see that the Content-Length is 16 bytes which is the exact length of the body. If the body had a trailing quote, that would be 17.

This is with version 3.2.1

**Comment by uglyog** [comment]
What version of pact are you using?

**Comment by SushmaShyam** [comment]
I was using version 2.1.9. I confirm that the issue is resolved by using version 3.2.1. Thanks.
